### ISSUE TITLE
Adding Httpmodule handler Error

### ISSUE THREAD

**Issue by leedavi** [40fingers/DNN-SEORedirect#14]
<add name="SeoRedirectModule" type="FortyFingers.SeoRedirect.Components.SeoRedirectModule,
40Fingers.DNN.Modules.SeoRedirect" preCondition="managedHandler" /> 

Seems to puts site in loop.

I originally thought this was lowercase issue, but even when the module it working and doing the correct redirect, adding this handler makes the site turn until timeout.

**Comment by Timo-Breumelhof** [comment]
We use it all the time and we don't have that issue, could it be you have redirects that redirect to each other?

**Comment by leedavi** [comment]
I checked that.   But it certainly looks like it, although it works OK from the module on the 404 page.  Maybe some setup I have that is not matching what should be.

Stephan said we can take a look together later in the week.

**Comment by Timo-Breumelhof** [comment]
ok, is it the first httpmodule line in web.config?

**Comment by leedavi** [comment]
Yes, I'll try and get time to debug it later this week.

**Comment by leedavi** [comment]
I've done a quick test on a local dev system and it all works OK.

Checking data, I think the problme is we have a url of:

... mentions-légales

redirecting to a url of:

... mentions-legales

So I think the http module is looping on that, because of the accent being ignored.

**Comment by Timo-Breumelhof** [comment]
Ok, are you doing the redirect mentions-légales > mentions-legales with our httpmodule?

**Comment by leedavi** [comment]
Yes, hence I think the loop is happening.  I've mentioned it to Stephan.

To solve it I simply renamed "mentions-legales" to "mentionslegales".

It was also uppercase "Mentions-légales", I think that might also be a loop condition.
